ext/dba/dba_gdbm.c

```diff
@@ -46,7 +46,9 @@ DBA_OPEN_FUNC(gdbm)
 	if(gmode == -1)
 		return FAILURE; /* not possible */
 
-	dbf = gdbm_open(info->path, 0, gmode, filemode, NULL);
+	/* TODO Use map_size parameter of dba_open() for block_size? */
+	/* TODO Add support for other GDBM flags support? See: https://www.gnu.org.ua/software/gdbm/manual/Open.html */
```

Comment:
Not sure if we should commit these TODO comments; they're likely just dead "code".